docs: add note on default branch for layers (#33919)
nuxt/nuxt: docs/1.getting-started/14.layers.md

})
```

::note

> This will fetch the main branch by default

Reviewer: ```suggestion
If a branch is not specified, this will clone `main`.
```
